src/OptimalControlProblems.jl

```diff
@@ -1,5 +1,50 @@
 module OptimalControlProblems
 
+using DataFrames
```

Comment:
I would remove `DataFrames`, I don't think we need it.

Reply:
I think we need it later for the meta here `const meta = DataFrame(infos .=> [Array{T}(undef, nb_problems) for T in types])`
It'll let us use `OptimalControlProblems.meta`

Reply:
Right, I think we should remove this line and depends only on the dictionaries in `OptimalControlProblems.meta`